### ISSUE TITLE
Order of entries in Quick Open View changed

### ISSUE THREAD

**Issue by ImJoeHs** [microsoft/vscode#139903]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.63.2
- OS Version: MacOS BigSur 11.6

Steps to Reproduce:

Ctrl + Q enter `workbench.action.quickOpenView`
![image](https://user-images.githubusercontent.com/13445155/147717516-55fb60cc-7336-4d91-8d0a-4215e03f1545.png)

Sidebar entries order in view do not match the real order in siderbar, and i can not find any setting to reorder that;

**Comment by TylerLeonhardt** [comment]
Looked into this a bit... what I was noticing is that if I reorder the sidebar entries, ctrl+Q was not being reflected based on that reorder (even if I reloaded the window).

It might be related to what @ImJoeHs is seeing... it might not...

I use this (well it looks like @sbatten touched this):
https://github.com/microsoft/vscode/blob/cdfee234401539007f33c87ae0c80b2b5304d381/src/vs/workbench/contrib/quickaccess/browser/viewQuickAccess.ts#L117

That returns a list that doesn't appear to honor the actual order.

@sbatten I'm suspicious of this commit but only because of the title "Large Refactorings Pane Composite Parts":
https://github.com/microsoft/vscode/commit/d4c222488de817d491f7c5686cf1161e9cceb7d4 any thoughts?

cc @bpasero

**Comment by sbatten** [comment]
I don't think this is a regression, at least not a recent one. If that list ever matched the order of the activity bar, it hasn't for a long time.

**Comment by TylerLeonhardt** [comment]
Is there a known way to get an accurate ordering of the side bar? Are you or @bpasero aware of anything like that?

**Comment by bpasero** [comment]
Maybe @sandy081

**Comment by TylerLeonhardt** [comment]
Thanks for tackling this, SteVen!
